src/core/src/CoreMain.py

```diff
@@ -101,6 +101,7 @@ def __init__(self, argv):
                     patch_installer.mark_installation_completed()
                     overall_patch_installation_operation_successful = True
                 self.update_patch_substatus_if_pending(patch_operation_requested, overall_patch_installation_operation_successful, patch_assessment_successful, configure_patching_successful, status_handler, composite_logger)
+            status_handler.log_truncated_packages()
 
```

Comment:
q: Wouldn't truncated packages be logged at the end of each status file write?

Reply:
@feng-j678 pending

Reply:
My opinion, this should be added for each status file write, post truncation, because that gives the details of what happened in each truncation (and thus status file writes). Will help in debugging issues in future. @kjohn-msft for more thoughts

Reply:
pending for koshy's input on this

Reply:
This comment is still open and needs feedback. @feng-j678 please make sure all existing comments are addressed